logging/log4j1_2/appenderSrc/test/resources/log4j.xml

```diff
@@ -26,11 +26,23 @@
     <appender name="test" class="com.microsoft.applicationinsights.log4j.v1_2.ApplicationInsightsAppender">
         <param name="instrumentationKey" value="c9341531-05ac-4d8c-972e-36e97601d5ff" />
```

Comment:
@littleaj is this real iKey? If yes please remove it asap!

Reply:
It's not, but I'm going to change it to make it more "mocked".